src/C4Config.cpp

```diff
@@ -497,12 +504,20 @@ bool C4Config::Save()
 			}
 			else
 			{
+#ifndef __linux__
 				filename.Copy(getenv("HOME"));
+#else
+				filename.Copy(Config.General.XDG_CONFIG_PATH);
```

Comment:
Why XDG_CONFIG_PATH? We might need a special ConfigPath for other platforms as well, so it shouldn't have a platform dependent name.